### ISSUE TITLE
BPRLoss

### ISSUE THREAD

**Issue by wuyueshijiu** [lazishu2000/CORONA#4]
作者你好，文章里提到了使用BPRLoss作为损失函数的一部分，开源代码我好像没找到这部分损失，恳请您解答，谢谢。

**Comment by lazishu2000** [comment]
感谢关注。目前代码中实现的部分选用infoNCE Loss (实际效果满足实验需求)，属于轻量级可替代实现，如需使用BPR Loss可直接使用torchrec中的官方实现：
from torchrec.losses.bpr import bpr_loss
loss = bpr_loss(pos_score, neg_score)

**Comment by MeltSky** [comment]
> 作者你好，文章里提到了使用BPRLoss作为损失函数的一部分，开源代码我好像没找到这部分损失，恳请您解答，谢谢。

您好，请问您成功复现这个实验了吗？

**Comment by wuyueshijiu** [comment]
> > 作者你好，文章里提到了使用BPRLoss作为损失函数的一部分，开源代码我好像没找到这部分损失，恳请您解答，谢谢。
> 
> 您好，请问您成功复现这个实验了吗？
没有 没实现这部分实验
